test/runtests.jl

```diff
@@ -64,9 +72,9 @@ cd(dirname(@__FILE__)) do
         error("Some tests exited with errors.")
     end
 
```

Comment:
I've thought about that, but I'm confused by the `is_unix` check below. Does it not work on windows?